docs/content/middlewares/overview.md

```diff
@@ -63,6 +63,45 @@ spec:
       - name: stripprefix
 ```
 
+```yaml tab="Kubernetes Ingress"
+# As a Kubernetes Ingress
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: middlewares.traefik.containo.us
+spec:
+  group: traefik.containo.us 
+  version: v1alpha1
+  names:
+    kind: Middleware
+    plural: middlewares
+    singular: middleware
+  scope: Namespaced
+
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: stripprefix
+  namespace: appspace
+spec:
+  stripPrefix:
+    prefixes:
+      - /stripit
+
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress
+  namespace: appspace
+  annotations:
+    # annotation the form <middleware-k8s-namespace>-<middleware-name>@kubernetescrd
```

Comment:
No worries, thanks for the changes :)

After a second thought, i think that the cross-provider example you added should be placed in the next examples section entitled "Referencing a Middleware from Another Provider".
It's more appropriate since the examples section in which you added it, is only showing how to declare and reference a middleware from one provider perspective.

I would also change the comment for the annotation to be more specific:
```suggestion
    # referencing a middleware from Kubernetes CRD provider: <middleware-namespace>-<middleware-name>@kubernetescrd
```

Do you think you can apply those changes too ?

Reply:
I fully agree that it would fit best in the section you mentioned, although i liked it in the other section, as it would provide a more complete overview - but, nonetheless, i moved the example and improved the comment with https://github.com/containous/traefik/pull/7269/commits/4143d12d3479b1f7d83b2834fe2234958f994b19.